apps/examples/wc-angular12/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "wc-angular12",
+    "version": "0.0.0",
+    "scripts": {
+        "ng": "ng",
+        "dev": "ng serve",
+        "build": "ng build",
+        "watch": "ng build --watch --configuration development"
+    },
+    "private": true,
+    "dependencies": {
+        "@angular/animations": "15.2.0",
+        "@angular/common": "15.2.0",
+        "@angular/compiler": "15.2.0",
+        "@angular/core": "15.2.0",
+        "@angular/forms": "15.2.0",
+        "@angular/platform-browser": "15.2.0",
+        "@angular/platform-browser-dynamic": "15.2.0",
+        "@angular/router": "15.2.0",
+        "@justeattakeaway/pie-button": "^0.10.2",
+        "@webcomponents/webcomponentsjs": "2.8.0",
```

Comment:
qq, why do we need this?

Reply:
I don't think we do 🤦 
It was from a part of a tutorial I don't think I needed